student.rb

```diff
@@ -5,10 +5,15 @@ class Student < Person
 
   def initialize(classroom, name = 'Unknown', age = nil, parent_permission: true)
     super(name, age, parent_permission: parent_permission)
+    assign_classroom(classroom)
```

Comment:
- [ ] In accordance to the project requirement and as raised earlier `@classroom` instance variable supposed to be initialized in the constructor. Konwing well that instance variables (those starting with @) are used to store object-specific data, and they are accessible throughout the instance's lifetime. Kindly compare the outpt of the screenshots below 👇 

<img width="521" alt="Untitled" src="https://github.com/ClintonYounge/School-Libraby-Project/assets/103386183/bab78912-19f6-4c6b-bbf2-6bdfcbb5c8d9">

<img width="511" alt="Untitled" src="https://github.com/ClintonYounge/School-Libraby-Project/assets/103386183/191b6a87-408e-4b36-979c-4f1f2510f890">

**If you implement it as suggested, you won't get duplicate values. As regards the linter, I ran it with the new implementation (@classroom=classroom) from here and it passed.** 👇 

<img width="528" alt="Untitled" src="https://github.com/ClintonYounge/School-Libraby-Project/assets/103386183/6bc3b811-76b4-4e7e-9d25-e96cc93b375f">

Reply:
@zieeco Thanks a lot. Very clear review.